src/posts/2021-08-09.md

```diff
@@ -0,0 +1,59 @@
+
+The new workflow automation that I created is a Slackbot for keeping track of these stories whose source data could otherwise fall through the cracks. Previously, someone would have to spend considerable time manually comparing the website and a spreadsheet to see who had not filled out the Google Form. Convenience is the commodity that always sells, so naturally there was a motivation to abstract some of the menial tasks in this process.
+
+Now every Friday morning, our serverless Lambda script self-activates, cross-references the website and submission data and gently reminds reporters of recent outstanding articles with a personalized direct message. (The name of the game is to make life easier for the staff, not harder, so verbal or technical feedback is very much appreciated.) The hope is that an extra nudge here and there will keep everyone honest in our undertaking. Of course, as I conceded to the staff when I announced this automation, it may not be a riveting time to go back and enter all the data, but the time is well worth it for the greater effort. In a sense, I think, it matters not what action you are carrying out as a journalist in the day-to-day, but for whom you are carrying out said action.
+
+My final lines of code for the summer were dedicated to our CMS, which is where website data is stored and staffers slate and publish content. In the CMS, we have a plugin called Yellow Lights, which, I learned, a former fellow created. The plugin won’t stop anyone from publishing their content, but it will run checks for style, disclosures, links to add and the like. My contribution to Yellow Lights adds a new panel to the interface and scans the body text for certain phrases that may warrant content warnings, suggesting additional copy to insert after the trailing paragraph of the text that is to be published. It may seem like a lot of code for a few sentences tacked onto the end of a piece, but an oft-heeded journalistic axiom is to [minimize harm](https://blogs.spjnetwork.org/ethicscode/?p=166) that a piece of content may inflict on some entity, typically an individual or group of people with “skin in the game,” so to speak. (The Society of Professional Journalists tends to provide more details on the ethics of these matters to those who seek to learn more.) In an effort to cover all our bases, adding a warning or disclaimer to certain instances of media that we publish is another preemptive measure to protect our audience, without whom we would be unable to spread our message.
+
```

Comment:
Same for this one, you could could grab a screenshot of yellow lights in the CMS with the new disclaimer